src/test/scala/mesosphere/mesos/TaskBuilderTest.scala

```diff
@@ -860,8 +856,8 @@ class TaskBuilderTest extends MarathonSpec {
     // The taskName is the elements of the path, reversed, and joined by dots
     assert("frontend.product" == taskInfo.getName)
     assert(2 == taskPorts.size)
-    assert(taskPorts.head == range.get.getBegin.toInt)
-    assert(taskPorts(1) == range.get.getEnd.toInt)
+    assert(taskPorts.contains(range.get.getBegin.toInt))
+    assert(taskPorts.contains(range.get.getEnd.toInt))
```

Comment:
You are not testing for the size of the range. 

Basically, if I understand it correctly, we want to test that the same ports are contained in the range and the taskPorts sequence. Can't we say that a bit more nicely?